frontend/src/pages/ApolloDemo.tsx

```diff
@@ -22,7 +23,9 @@ const ApolloDemoPage = () => {
    *    - error: a ApolloError object with some details, in case things fail
    *    - data: the data fetched by your query
    */
-  const { data, loading, error } = useQuery(GROUPS_QUERY)
+  const { data, loading, error } = useQuery(GROUPS_QUERY, {
+    pollInterval: 5000,
+  })
```

Comment:
I noticed this kept going after navigating away from the page, any idea how to not do that?

Reply:
The `useQuery` hook returns a `stopPolling` method that should be called when the component unmounts. I'd do it like this:

```jsx
const { data, loading, error, stopPolling } = useQuery(...)

useEffect(function cleanupOnUnmount() {
  // When the component unmounts, the return value of this method will be called
  return stopPolling
}, [])
```